Sources/MapboxDirections/Directions.swift

```diff
@@ -141,6 +141,7 @@ open class Directions: NSObject {
     
     // MARK: Getting Directions
     
+    internal typealias RouteCompletionHandlerInteral = (_ waypoints: [Waypoint]?, _ routes: [Route]?, _ error: DirectionsServiceError?) -> Void
```

Comment:
Not sure what needs to be internal about this. Isn’t DirectionsServiceError already public?